skyportal/models/obj.py

```diff
@@ -215,6 +216,8 @@ class Obj(Base, conesearch_alchemy.Point):
         sa.ARRAY(sa.String), nullable=True, doc="Alternative names for this object."
     )
 
+    healpix = sa.Column(healpix_alchemy.Point, index=True)
```

Comment:
```suggestion
    healpix = sa.Column(healpix_alchemy.Point, index=True, nullable=False)
```

Reply:
@acrellin has examples of objects without RAs and Decs (they are just None), so I think this needs to be nullable.

Reply:
Why?

Reply:
If RA and Dec is not defined, healpix is just None right?